Scripts/DCS-BIOS/lib/common/Logger.lua

```diff
@@ -53,7 +53,7 @@ function Logger:log(level, obj)
 
 	if type(obj) == "table" then
 		self:log_simple(level, "Logging table, consider using setting max_bytes_to_log and call log_table() directly.")
-		self:log_table(level, obj, 10)
+		self:log_table(level, obj, 40)
```

Comment:
Is there any way we can distinguish an array from a table? Guessing not in lua...

Reply:
Well this is the problem I was having, I tried to log the parse_indication with the original function and nothing came out. I guess I have to do some local testing.

Reply:
The only result of parse_indication which has an array is the KA-50 iirc (I may have removed that too, I can't recall atm)